client/containers/HomePage.js

```diff
@@ -11,19 +11,32 @@ import Condition from 'components/Condition/Condition';
 class HomePage extends Component {
   handleOnClick = () => {
     this.props.fetchPlaces(this.props.condition);
-  }
+  };
 
   handleOnConditionChange = (value) => {
     this.props.setRadius(value);
-  }
+  };
+
   render() {
     const { condition, place } = this.props;
+    const { searchEnabled } = place;
```

Comment:
yes, searchEnabled is the concluded when we start an async call and received request, it is different from latlng as lat lng are part of the argument of the async call.. fyi, I use searchEnable to disable button when we do ajax. I don't know much about redux-saga and I use it as a practice, not really part of Task 1

Reply:
disabling button matters as when user repeatedly clicking the search button even the initial request is not yet finish

Reply:
we normally try to avoid double logic like this, i.e. a generic condition plus a specific condition. If you really want to use a generic condition, try to move your lat lng inside your searchenabled condition setting, this will be a much more clearer approach

Reply:
this is well noted, i'll apply it